sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuTransitionOverrides.scala

```diff
@@ -480,4 +502,25 @@ object GpuTransitionOverrides {
       case _ => plan
     }
   }
+
+  /** Return list of matching predicates present in the plan */
+  def findOperators(plan: SparkPlan, predicate: SparkPlan => Boolean): Seq[SparkPlan] = {
```

Comment:
@andygrove I think you wrote this code originally, any caveats with this that you are aware of with the AQE stuff?

Reply:
This looks good.